static/style/stylesheet.css

```diff
@@ -60,8 +70,9 @@
     border: 0;
 }
 .tfclear{
-    clear:both;
+    float: left;
```

Comment:
If this isn't going to clear anymore, we probably shouldn't call it "tfclear" anymore :)